react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```

Comment:
Although it works as an empty string it is best to put a `#` as the `href` for the "Home" route. It shows intention and that you didn't accidentally leave it empty.
```JSX
<a href="#" className="navbar-brand">
```